lib/vsc/utils/fancylogger.py

```diff
@@ -431,7 +438,7 @@ def getRootLoggerName():
         return "not available in optimized mode"
 
 
-def logToScreen(enable=True, handler=None, name=None, stdout=False):
+def logToScreen(enable=True, handler=None, name=None, stdout=False, color=False):
```

Comment:
Since there are three cases, I would go with strings:

* `always`: always use color, even when the output device is not a terminal
* `yes` (or `auto`): color (only for terminal output device)
* `no`: no color